src/services/JobService.ts

```diff
@@ -91,5 +91,26 @@ export async function killJobs(jobs: CompactJob[], userEmail: string) {
   if (error) {
     const message = error.reason as string;
     throw new Error(`Failed to kill job due to ${message}`);
+  } else {
+    refresh();
```

Comment:
Same comment as the other PR. I think refreshing should not be the responsibility of killJobs. It should just kill jobs.  You can do that outside this method in the upper calling method.